src/components/shared/styles/common.scss

```diff
@@ -41,3 +41,14 @@ $leafletControlBackgroundHover: rgba(255,255,255,0.5);
 
 $sidebarNarrow: 400px;
 $sidebarWide: 500px;
+
+%soloControl {
```

Comment:
I wouldn't put this in common.css because it is too specific.

If "copy-pasted" css is a problem, make a "mapControl.scss" file for those mapControl buttons. You could have both fullscreenControl.scss and mapControl.scss or just mapControl.scss. Imo here won't be that many styles so only a mapControl.scss shouldn't be a problem.